cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/monitor/sampling/MetricFetcher.java

```diff
@@ -96,7 +128,17 @@ protected void fetchMetricsForAssignedPartitions() throws SamplingException {
     try {
       MetricSampler.Samples samples = fetchSamples();
       if (_sampleStore != null) {
-        _sampleStore.storeSamples(samples);
+        if (_samplingMode == MetricSampler.SamplingMode.ONGOING_EXECUTION) {
+          _sampleStore.storeSamples(new MetricSampler.Samples(Collections.emptySet(), samples.brokerMetricSamples()));
+        } else {
+          _sampleStore.storeSamples(samples);
+        }
+      }
+
+      if (_samplingMode == MetricSampler.SamplingMode.ONGOING_EXECUTION &&
+          _sampleStoreForPartitionMetricsDuringExecution != null) {
```

Comment:
Nit: The common convention is to carry `&&` to the new line. -- i.e. break before an operator.

This should be a CheckStyle rule -- I created to enforce that: https://github.com/linkedin/cruise-control/issues/1549